frontend/src/reviews/ReviewCommentDisplay.tsx

```diff
@@ -49,16 +83,33 @@ export default function ReviewCommentDisplay({ response, onReplyButtonClick }: R
               </IconButton>
             </Stack>
           </Stack>
-          {response.comment && (
-            <div>
-              <Divider sx={{ mt: 1, mb: 2 }} />
+          <Divider sx={{ mt: 1, mb: 2 }} />
+          {response.comment && !isEditMode && (
+            <Stack spacing={2}>
               <MarkdownDisplay>{response.comment}</MarkdownDisplay>
-            </div>
+              {response.updatedAt !== response.createdAt && (
+                <Typography variant='caption' sx={{ fontStyle: 'italic' }}>
+                  Updated {formatDateTimeString(response.updatedAt)}
+                </Typography>
+              )}
+            </Stack>
+          )}
+          {isEditMode && (
+            <>
+              <RichTextEditor value={updatedComment} onChange={(input) => setUpdatedComment(input)} />
+              <Box sx={{ float: 'right' }}>
+                <Button onClick={handleEditOnSave}>Save</Button>
```

Comment:
We should also have a cancel button

Reply:
Duh this would be so useful no idea why I didn't add it! :P